README.md

```diff
@@ -1,99 +1,104 @@
 # LBS-server
 
 # Installation Guide
+
 ## Build and Run
-### Step 1. Clone the github repo to your prefered directory using
-```
-git clone https://github.com/Suwi-inc/LBS-server.git
-``` 
-### Step 2. Inside the repository directory install the project dependencies
+
+### Step 1. Clone the github repo to your preferred directory
 ```sh
-poetry install
+git clone https://github.com/Suwi-inc/LBS-server.git
+cd LBS-server
 ```
-### Step 3: Create .env
 
-The project depends on some config that it expects to be provided from a .env file at the `LBS` directory. 
+### Step 2. Install Docker if it is not installed
+https://docs.docker.com/engine/install/
+
+### Step 3: Create .env file
+
+The project depends on some config that it expects to be provided from an .env file at the `LBS` directory.
 ```sh
 cd LBS
-``` 
-Here is the expected content of the file
-```env
-DB_URI='Postgres_DB_connect_string_here'
-SECRET_KEY='your_auth_secret'
 ```
-### Step 4: Run Migrations
-Inside the `LBS` directory run
+
+Here is the expected content of the file:
 ```sh
-poetry run flask db init
-poetry run flask db migrate
-poetry run flask db upgrade
+SECRET_KEY='your_auth_secret'
 ```
-### Step 5: Run the application
-Within the `LBS` directory
+
+# Step 4: Define environment variables
 ```sh
-poetry run flask --app app run   
+export POSTGRES_USER=postgres
+export POSTGRES_PASSWORD=pass
+export POSTGRES_DB=db
+export SECRET_KEY=your_auth_secret
+export REMOTE_DB_PORT=5433
 ```
-# Build and Run on Docker
-Required docker version: >=4.25.2
-Before running, place an .env file in the root directory
+
+### Step 5: Run docker compose
+Run this command when you update the server
 ```sh
 docker compose up --build -d
 ```
-# Project File structure 
+
+### Step 6: Stop docker compose
```

Comment:
Stopping docker compose is not a part of running the project

Reply:
Changed